compiler/plugins/target/AMD-AIE/iree-amd-aie/Transforms/AMDAIEAddLoweringStrategy.cpp

```diff
@@ -23,6 +25,7 @@ class AMDAIELoweringStrategyPass
  public:
   void getDependentDialects(DialectRegistry &registry) const override {
     registry.insert<
+        AMDAIE::AMDAIEDialect, IREE::Codegen::IREECodegenDialect,
```

Comment:
Why do you have to add these? I don't see what you added has to use these?

Reply:
I added a new lit test file which necessitated their addition.

We never had any lit test that was verifying the tile/pack config - and since the lit test invokes the lowering strategy pass as a standalone pass, the MLIRContext doesn't have the required Dialects' info to form the attributes which KernelDispatch forms.

This wasn't required previously because we were using some pass pipeline (bigger picture/context) to test it - it gets loaded in MLIRContext by at least one of the passes being used in that pass pipeline we use.